endpoints/main.go

```diff
@@ -28,17 +33,28 @@ var totalFailedUptimeChecksCounterVec *prometheus.CounterVec
 // Record the metrics into the appropriate prometheus register under the label specified by organizationName
 // recordLongRunningMetrics specifies wether or not to record information contained in the capability statment
 func getHTTPRequestTiming(urlString string, organizationName string, recordLongRunningMetrics bool) {
-	var resp, responeTime, err = querier.GetResponseAndTiming(urlString)
+	ctx := context.Background()
+	// Drop connection if no reply within 30 seconds
```

Comment:
this is doing more than dropping the connection if no reply, right? it's basically cleaning everything up if things don't finish within 30 seconds.

Maybe: "Closing context if HTTP request and response processing is not completed within 30 seconds. This includes dropping the request connection if there's no reply within 30 seconds."